templates/galaxy/config/global_host_filters.py.j2

```diff
@@ -59,7 +59,7 @@ DOMAIN_SECTIONS = {
     'hicexplorer': GENERAL_NGS_SECTIONS + ["hicexplorer", "graph_display_data", "peak_calling"],
     'virology': GENERAL_NGS_SECTIONS + ["assembly", "annotation", "phylogenetics"],
     'nanopore': GENERAL_NGS_SECTIONS + ["nanopore", "ncbi_blast", "fasta_fastq", "assembly", "graph_display_data", "metagenomic_analysis"],
-    'ecology': ["join__subtract_and_group", "convert_formats", "operate_on_genomic_intervals", "statistics", "graph_display_data", "machine_learning",
+    'ecology': ["join__subtract_and_group", "convert_formats", "Data and Metadata Management", "operate_on_genomic_intervals", "statistics", "graph_display_data", "machine_learning",
```

Comment:
It needs to be the id, not the name. You will find it in the tool_conf.xml file.